contents/customers/ycombinator.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How Y Combinator uses PostHog experiments to boost engagement by 40%
```

Comment:
Should this be 'used' rather than 'uses'? I don't have a strong feeling about it, it just parses more easily.

Reply:
```suggestion
title: How Y Combinator used PostHog experiments to boost engagement by 40%
```